remoteworkitem/remoteworkitem_test.go

```diff
@@ -80,7 +81,7 @@ func TestGitHubIssueMapping(t *testing.T) {
 	}
 
 	workItemMap := WorkItemKeyMaps[ProviderGithub]
-	remoteWorkItem := RemoteWorkItem{ID: "xyz", Content: []byte(content)}
+	remoteWorkItem := TrackerItem{Item: string(content[:]), RemoteItemID: "xyz", BatchID: "bxyz", TrackerQueryID: uint64(0)}
```

Comment:
Not to be picky, but generally, if you have to rename the variable because the type changed, it had a poor name to begin with. A name should express the use/concept, not the language Type system.